activestorage/lib/active_storage/transformers/image_processing_transformer.rb

```diff
@@ -1,45 +0,0 @@
-# frozen_string_literal: true
-
-begin
-  require "image_processing"
-rescue LoadError
-  raise LoadError, <<~ERROR.squish
-    Generating image variants require the image_processing gem.
```

Comment:
I think we still need in the new require in case people don't have the gem in their Gemfile. In the case people didn't uncommented it.

Reply:
Restored it.